### DISCUSSION TITLE
[Question][Hw3] Cannot find example1.ll

### DISCUSSION THREAD

**Discussion by jirheee** [prosyslab-classroom/cs348-information-security#519] [Q&A]
I am using docker image provided.

I tried to run "test/example1.ll" as announced for test, but there was no such file.

What should I do to do this operation?

**Comment by RiceBiscuits** [comment]
You should generate example1.ll by executing "make" command under the test directory or using clang.

**Comment by jirheee** [comment]
Thanks!
